manifests/profile/apt_mirror.pp

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2021 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::apt_mirror
+#
+# Profile for apt repo mirror
+#
+# @example
+#   include nebula::profile::apt_mirror
+class nebula::profile::apt_mirror (
+) {
+
+  package {
+    [ 'debmirror', 'debian-keyring', 's3fs' ]:
+  }
```

Comment:
At least debian-keyring and s3fs are dependencies, but have very little to do with making an apt mirror, so we should use `ensure_packages` here to avoid incidental conflicts in other cases where we might use them.

Reply:
Changed this.